bindata/bootkube/manifests/00_etcd-platform-configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: openshift-platform
+  namespace: openshift-etcd
+data:
+  platform: "{{ .Platform }}"
```

Comment:
does this data exist already? I think MCO has access to it can we check

Reply:
yes it does, quick search revealed

```
apiVersion: v1
items:
- apiVersion: config.openshift.io/v1
  kind: Infrastructure
  metadata:
    name: cluster
  spec:
...
  status:
   ...
    platform: AWS
    platformStatus:
      aws:
        region: us-west-1
      type: AWS
```